### DISCUSSION TITLE
Replace .reply() with .respond() and add actual reply functionality to CommandEvent?

### DISCUSSION THREAD

**Discussion by Chew** [Chew/JDA-Chewtils#42] [Ideas]
(We should really get those fancy issue templates).

If we replaced .reply() with actually replying to the message, would this be a breaking change? Would it be worth it to make?

Would we need to implement the extra 14 or whatever methods to support replying? Discussion!

**Comment by Andre601** [comment]
I feel like this would be a good change.
Also, just make it a new major version imo. That way should people know that it contains breaking changes, and if not is it their fault...

I think something like `respond(String, boolean)` where the boolean sets if the response is actually a reply (Should quote the message) would be an idea.
Other idea I can think of is `respond(String).withReply(true)` but not sure how doable this would be since the current reply function is just a void executing a sendMessage, no?
